src/kd_time_main.cpp

```diff
@@ -0,0 +1,28 @@
+#include "polyhedralGravity/input/YAMLConfigReader.h"
+#include <benchmark/benchmark.h>
```

Comment:
You don't consistently include everything you need (yeah, I know transitivity,.. but this not always the case 🙂). In this file, you use e.g. the `Polyhedron` --> include this file

Reply:
@schuhmaj I've integrated the tool include-what-you-use and produced output that solves the problem, but also finds issues in files not part of the KDTree and tends to move library includes into cpp files. Should I ignore these suggestions and just fix my own header files and keep source files and higher level code untouched or is there more to it?